openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -79,7 +79,7 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                 class="subsection-text outline-button"
                                 id="${ subsection['id'] }"
                             >
-                            % if num_graded_problems and not (subsection.get('format') or 'special_exam_info' in subsection):
+                            % if graded and scored and 'special_exam_info' not in subsection:
```

Comment:
The first bit (`num_graded_problems` -> `graded and scored`) is so that even if we don't have a precise count of the problems, we still show the graded icon. (i.e. for LTI modules)

The second bit (dropping checking `format`) is because as best as I can tell (or Nick can remember), we don't seem to really need to check that -- that controls whether there is a custom subtitle. Doesn't seem related to whether we should show a graded icon.